ext/standard/array.c

```diff
@@ -243,7 +243,7 @@ static zend_always_inline int php_array_key_compare_string_case_unstable_i(Bucke
 		l2 = s->key->len;
 	} else {
 		s2 = zend_print_long_to_buf(buf2 + sizeof(buf2) - 1, s->h);
-		l2 = buf2 + sizeof(buf2) - 1 - s1;
+		l2 = buf2 + sizeof(buf2) - 1 - s2;
```

Comment:
This does look off, however it seems `php_array_key_compare_string_case_unstable_i` is unused and I also couldn't find any [references on Sourcegraph](https://sourcegraph.com/search?q=context:global+php_array_key_compare_string_case_unstable_i&patternType=literal), so I wonder if we should just remove it.